src/js/stores/__tests__/MesosStateStore-test.js

```diff
@@ -142,12 +142,22 @@ describe('MesosStateStore', function () {
 
     it('should find a currently running task', function () {
       var result = MesosStateStore.getTaskFromTaskID(1);
-      expect(result).toEqual({id: 1});
+      expect(result).toEqual({
```

Comment:
You can also do `expect(result.get()).toEqual({id: 1});` which is a little less work for yourself 😃